release-notes/2023-10/MongoDB_Migration/README.md

```diff
@@ -0,0 +1,49 @@
+# Existing MongoDB Data Migration to a Single Instance
+
+## Do you need to worry about what is in this folder?
+
+The content in this folder can help you if you have recognized a need to migrate existing SystemLink Enterprise MongoDB data to a single MongoDB instance. You should not need this content if deploying a new installation of SystemLink Enterprise.
+
+## Background
+
+Prior to version 2023-10 installations of SystemLink Enterprise deployed a MongoDB instance per service. In version 2023-10 the services were refactored to use a global MongoDB connection string allowing customers to go from running over twenty (20) different MongoDB instances down to one.
+
+Controlling this functionality is done through global Helm values. Configure `global.mongodb.install` to `false` to enable this feature. You will also need to configure `global.secrets.mongodb.connection_string`. Refer to [Configuring Systemlink Enterprise](https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/config-systemlink-enterprise.html) for more information.
+
+## Prerequisites
```

Comment:
I would put this info before we send them to the process doc to avoid them running off and missing important info

Reply:
Do you mean move "Prerequisites" before "Process"?

Reply:
Yeah

Reply:
I didn't like the "Process" section so I moved some of that content into "Do you need to worry about what is in this folder?" and put the link to ni.com inside a new "Enable a Single Instance" section. Let me know if that flows better.

Reply:
That works for me